client/cmd/dexc/config.go

```diff
@@ -54,18 +57,37 @@ func setNet(applicationDirectory, net string) string {
 	return filepath.Join(netDirectory, "dexc.db")
 }
 
+// defaultHostByNetwork accepts configured host with configured network
+// flags and returns the network specific default host if the provided host is
+// empty
+func defaultHostByNetwork(host string, testnet, simnet bool) string {
```

Comment:
How about `net dex.Network` instead of `testnet, simnet bool`?

Reply:
Yep